Source/WebCore/html/RadioInputType.cpp

```diff
@@ -136,7 +136,7 @@ auto RadioInputType::handleKeydownEvent(KeyboardEvent& event) -> ShouldCallBaseE
     // Tested in WinIE, and even for RTL, left still means previous radio button (and so moves
     // to the right).  Seems strange, but we'll match it.
     // However, when using Spatial Navigation, we need to be able to navigate without changing the selection.
-    if (isSpatialNavigationEnabled(element->document().frame()))
+    if (isSpatialNavigationEnabled(element->document().protectedFrame().get()))
```

Comment:
This is silly. We directly check `settings().spatialNavigationEnabled()` and call it a day.

Reply:
addressed in https://github.com/WebKit/WebKit/pull/42512